src/richie/apps/persons/templates/persons/cms/person_detail.html

```diff
@@ -1,19 +1,24 @@
 {% extends "richie/fullwidth.html" %} {% load cms_tags i18n %} {% block content %}
 <div class="person-detail">
+  <h1 class="person-detail__title">{{ current_page.get_title }}</h1>
 
-    <h1 class="person-detail__title">{{ current_page.get_title }}</h1>
-
+  <div class="person-detail__card">
```

Comment:
is card the right terminology for the detail page?

Reply:
Well, since it's a card, yeah. Note than for futur related content in this page i plan to put them in another container, not in the card.